readthedocs/templates/builds/build_detail.html

```diff
@@ -69,6 +69,23 @@
           </a>
         </li>
       </div>
+
+
+      {# Show rebuild button only if the version is external and it's the latest build for this version #}
+      {# see https://github.com/readthedocs/readthedocs.org/pull/6995#issuecomment-852918969 #}
+      {% if request.user|is_admin:project and build.version.type == "external" and is_latest_build %}
```

Comment:
Not related to this PR, but I remember `is_admin: project` being expensive, we are using the check 3 times now. We can move it to the context instead.

https://github.com/readthedocs/readthedocs.org/blob/0340604163a15fccc8b786ce0a5e589bf78f6f24/readthedocs/projects/views/public.py#L129-L132